packages/react-components/src/components/VideoGallery/LargeGalleryLayout.tsx

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { useMemo, useRef, useState } from 'react';
+import { isNarrowWidth } from '../utils/responsive';
+/* @conditional-compile-remove(gallery-layouts) */
+import { isShortHeight } from '../utils/responsive';
+import { LayoutProps } from './Layout';
+import { OverflowGallery } from './OverflowGallery';
+import { GridLayout } from '../GridLayout';
+import { Stack } from '@fluentui/react';
+import { useOrganizedParticipants } from './utils/videoGalleryLayoutUtils';
+import { rootLayoutStyle } from './styles/DefaultLayout.styles';
+import { videoGalleryLayoutGap } from './styles/Layout.styles';
+/* @conditional-compile-remove(gallery-layouts) */
+import { VERTICAL_GALLERY_TILE_SIZE_REM } from './styles/VideoGalleryResponsiveVerticalGallery.styles';
+
+/**
+ * Props for {@link LargeGalleryLayout}.
+ *
+ * @private
+ */
+export type LargeGalleryProps = LayoutProps;
+
+const DEFAULT_CHILDREN_PER_PAGE = 5;
+/* @conditional-compile-remove(gallery-layouts) */
+const REM_TO_PIXEL = 16;
```

Comment:
Could you use _convertRemToPx instead? Note: you will need to mock this function for any unit tests like [this](https://github.com/Azure/communication-ui-library/blob/70626a43d2c7d7a7a044cf366b16ce7a07b155f5/packages/react-components/src/components/ResponsiveVerticalGallery.test.tsx#L126)